parser/src/Parse/Expression.hs

```diff
@@ -191,11 +191,18 @@ ifExpr elmVersion =
         >> whitespace
   in
     do
-      first <- ifClause elmVersion
-      rest <- many (try $ (,) <$> elseKeyword <*> ifClause elmVersion)
-      final <- (,) <$> elseKeyword <*> expr elmVersion
+      (first, firstMultiline) <- trackNewline $ ifClause elmVersion
+      (rest, restMultiline) <- trackNewline $ many (try $ (,) <$> elseKeyword <*> ifClause elmVersion)
+      (final, finalMultiline) <- trackNewline $ (,) <$> elseKeyword <*> expr elmVersion
 
-      return $ E.If first rest final
+
+      return $ E.If first rest final $
+          case (firstMultiline, restMultiline, finalMultiline) of
+            (JoinAll, JoinAll, JoinAll) ->
+                multilineToBool JoinAll
+
+            _ ->
+                multilineToBool SplitAll
```

Comment:
Ah, Bool was the old way of storing multiline... the new arg to `If` should be `Multiline` (from AST.V0_16) instead of `Bool`

Reply:
Changed to `Multiline`, I used a `Bool` at first because `Multiline` was not yet used in `parser/src/AST/Expression.hs`, which made me suspiscious of using it.

Note that `multilineToBool` in `parser/src/Parse/Helpers.hs` seems redundant with `isMultiline` from `parser/src/AST/V0_16.hs`, but both may be needed for some reason I did not understand.